db/migrate/20230406091451_create_posts.rb

```diff
@@ -0,0 +1,17 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      add_foreign_key :posts, :users, column: :author_id
```

Comment:
- [ ] the foreign key  isn't added in the schema file  to make things simple for you please try to use `t.references :author, foreign_key: { to_table: 'users' }` instead of adding the column and then adding the foreign key

Reply:
@youmari , Thanks a lot! Changes implemented.